shortcodes/js/components/AverageLabelTime.js

```diff
@@ -71,6 +72,20 @@ class AverageLabelTime extends Component {
 			labels.push( new Date( recordDate * 1000 ).toDateString() );
 			issuesNumData.push( Object.values( recordedIssues ).length );
 		} );
+
+		// Will not display a chart.
+		if ( numOnly ) {
+			const currentAvgTimeIndex = avgTimeData.length -1,
```

Comment:
Space is missing after `-` sign. 
```suggestion
			const currentAvgTimeIndex = avgTimeData.length - 1,
```